packages/plugins/users-permissions/admin/src/pages/Roles/EditPage/tests/index.test.js

```diff
@@ -19,25 +21,20 @@ jest.mock('../../../../hooks', () => {
 
   return {
     ...originalModule,
-    useFetchRole: id => {
-      const role = {
-        id,
-        name: 'Authenticated',
-        description: 'Default role given to authenticated user.',
-        type: 'authenticated',
-      };
-      const onSubmitSucceed = jest.fn();
-
-      return { role, onSubmitSucceed };
-    },
+    usePlugins: () => ({
+      ...originalModule.usePlugins,
+      isLoading: false,
+    }),
   };
 });
 
-it('renders users-permissions edit role and matches snapshot', () => {
+function makeAndRenderApp() {
   const history = createMemoryHistory();
-
+  const messages = {
```

Comment:
`const messages = { en: {...}}`

Reply:
Not sure why, but with Jest the error wouldn't display properly when I put the messages in an `en` property.

So I updated it to do what I saw in several parts of the codebase, which is passing an empty messages object:

```jsx
<IntlProvider locale="en" messages={{ en: {} }} textComponent="span">
```

and use the defaultMessage for testing